<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tic_tac_toe</class>
 <widget class="QMainWindow" name="Tic_tac_toe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>275</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>275</width>
    <height>400</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>275</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Tic_tac_toe</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/icon/image/tic-tac-toe_icon.png</normaloff>:/icon/image/tic-tac-toe_icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>234</width>
      <height>377</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QComboBox" name="levelBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="currentText">
          <string>Easy</string>
         </property>
         <property name="minimumContentsLength">
          <number>2</number>
         </property>
         <item>
          <property name="text">
           <string>Easy</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Hard</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Select a character</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QRadioButton" name="rbX">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 170, 0);</string>
           </property>
           <property name="text">
            <string>X</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbO">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(170, 0, 0);</string>
           </property>
           <property name="text">
            <string>O</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetNoConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item row="0" column="2">
        <widget class="QPushButton" name="btn9">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>22</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(89, 153, 255);
border-color: rgb(21, 21, 21);
color: rgb(170, 0, 0);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="btn3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>22</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(89, 153, 255);
border-color: rgb(21, 21, 21);
color: rgb(170, 0, 0);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btn8">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>22</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(89, 153, 255);
border-color: rgb(21, 21, 21);
color: rgb(170, 0, 0);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="btn4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>22</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(89, 153, 255);
border-color: rgb(21, 21, 21);
color: rgb(170, 0, 0);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="btn1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>22</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(89, 153, 255);
border-color: rgb(21, 21, 21);
color: rgb(170, 0, 0);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="btn6">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>22</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(89, 153, 255);
border-color: rgb(21, 21, 21);
color: rgb(170, 0, 0);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="btn7">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>22</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(89, 153, 255);
border-color: rgb(21, 21, 21);
color: rgb(170, 0, 0);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="btn5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>22</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(89, 153, 255);
border-color: rgb(21, 21, 21);
color: rgb(170, 0, 0);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="btn2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>22</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(89, 153, 255);
border-color: rgb(21, 21, 21);
color: rgb(170, 0, 0);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="ShowResult">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(170, 0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btnStart">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(60, 255, 57);</string>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>18</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnClose">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 16, 28);</string>
         </property>
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>Tic_tac_toe</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>349</y>
    </hint>
    <hint type="destinationlabel">
     <x>36</x>
     <y>376</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
